.ci/azure-pipelines.yml

```
@@ -0,0 +1,98 @@
+trigger:
+  branches:
+    include:
+    - refs/tags/*
+
+jobs:
+  - job: build
+    displayName: Build
+    pool:
+      vmImage: "ubuntu-latest"
+    steps:
+      - checkout: self
+        clean: true
+        submodules: true
+        persistCredentials: true
+
+      - task: CmdLine@2
+        displayName: Setup directories and variables
+        inputs:
+          script: |
+            echo "##vso[task.setvariable variable=TIZEN_HOME_DIR]$(Agent.TempDirectory)/tizen"
+            echo "##vso[task.setvariable variable=CERTIFICATE_DIR]$(Agent.TempDirectory)/certificates"
+          failOnStderr: false
+
+      - task: UseNode@1
+        displayName: Install Node
+        inputs:
+          version: '12.x'
+
+      - task: CmdLine@2
+        displayName: Install Jellyfin Tizen
+        inputs:
+          script: | 
+            yarn install
+            JELLYFIN_WEB_DIR=./node_modules/jellyfin-web/dist yarn build
+          failOnStderr: false
+
+      - task: CmdLine@2
+        displayName: Setup Tizen Studio
+        inputs:
+          script: | 
+            export TIZEN_CLI_INSTALLER_URL=http://download.tizen.org/sdk/Installer/tizen-studio_3.7/web-cli_Tizen_Studio_3.7_ubuntu-64.bin
+            export TIZEN_STUDIO=$TIZEN_HOME_DIR/tizen-studio
+            
+            mkdir -p $TIZEN_HOME_DIR
+            cd $TIZEN_HOME_DIR
+
+            INSTALLER=$(Agent.TempDirectory)/$(basename "$TIZEN_CLI_INSTALLER_URL") \
+              && curl "$TIZEN_CLI_INSTALLER_URL" > "$INSTALLER" \
+              && chmod 755 "$INSTALLER" \
+              && "$INSTALLER" --no-java-check --accept-license $TIZEN_STUDIO \
```

Why not put these all on separate lines?

---

You're correct, these could just be separate lines. It's a remnant from my initial setup which used a docker container. Docker creates a layer per command/line, so it makes more sense to combine lines in order to reduce overhead.

We're not using Docker anymore, so I could change it to this:
```suggestion
    export INSTALLER=$(Agent.TempDirectory)/$(basename "$TIZEN_CLI_INSTALLER_URL")       
    curl "$TIZEN_CLI_INSTALLER_URL" > "$INSTALLER"
    chmod 755 "$INSTALLER"
    "$INSTALLER" --no-java-check --accept-license $TIZEN_STUDIO
```

---

Docker is definitely weird :) that's what I figured, but separate lines everywhere would be best in this file.

---

I've separated the lines